entities/board.py

```diff
@@ -29,6 +29,7 @@ def __init__(self) -> None:
         # Stores mapping from unique pairs of (piece_type, colour)
         # to a set of positions.
         # Used for getting all positions for a specific piece type.
+        # TODO (@EK): think about it in case of pawn promotion
```

Comment:
Let's follow Google Python guidelines.
https://google.github.io/styleguide/pyguide.html#312-todo-comments

Here it would become:
`# TODO(@cardinal-evgeniy): think about it in case of the pawn promotion`